interfaces/builtin/x11.go

```diff
@@ -71,6 +71,9 @@ unix (bind, listen, accept)
 unix (bind, listen, accept)
      type=stream
      addr="@/tmp/.ICE-unix/[0-9]*",
+unix (bind, listen, accept)
+     type=dgram
+     addr="@nvidia[0-9a-f]*",
```

Comment:
This is in x11PermanentSlotAppArmor. Why does an X server provide a listening service on an abstract socket with @nvidia? This feels out of place here. Can you comment?

Reply:
The socket is created by Xorg when loading the nvidia driver. It is then used by open GL applications (see client rule in opengl interface: https://github.com/snapcore/snapd/blob/master/interfaces/builtin/opengl.go#L68). Without this rule, Xorg refuses to start, that is why I included it here - but there are reasons too to move this to the opengl interface. Please let me know which is your preference.

Reply:
Based on your feedback, I think it is reasonable to keep it in the x11 slot policy since most applications won't need it. Please add a comment above this rule on why you added it. I suggest:

```
# On systems with Tegra drivers, X11 needs to create the socket for clients to
# use.
```

Reply:
Comment added now